src/frontend/komponenter/Fagsak/Behandlingskort/Behandlingskort.tsx

```diff
@@ -136,11 +136,15 @@ const Behandlingskort: React.FC<IBehandlingskortProps> = ({ åpenBehandling }) =
                 informasjon={[
                     {
                         label: 'Opprettet',
-                        tekst: formaterIsoDato(åpenBehandling.opprettetTidspunkt, datoformat.DATO),
+                        tekst: formaterIsoDato(åpenBehandling.opprettetTidspunkt, Datoformat.DATO),
                     },
                     {
                         label: 'Vedtaksdato',
-                        tekst: formaterIverksattDato(åpenBehandling.vedtak?.vedtaksdato),
+                        tekst: formaterIsoDato(
+                            åpenBehandling.vedtak?.vedtaksdato,
+                            Datoformat.DATO,
+                            'Ikke satt'
```

Comment:
Det er kanskje en del jobb, så vi trenger ikke å ta det i denne PRen, men `formaterIsoDato` burde ta inn et objekt slik at vi kan få navn på inputvariablene. 

Tror det generelt er en grei regel for alle funksjoner i JS at man burde sende inn et objekt og ikke parametere direkte

Reply:
Enig i det! Skal ta en ny titt på hele datoflyten etter at jeg har oppgradert datovelgerne (for da er nok mye overflødig, som forhåpentligvis denne funksjonen), så venter med å gjøre noe enn så lenge